variables.tf

```diff
@@ -85,3 +85,15 @@ variable "post_cilium_install_script" {
   description = "A script to be run right after installing Cilium."
   type        = string
 }
+
+variable "total_control_plane_nodes" {
+  default     = 3
```

Comment:
will setting a default here break EKS (where you don't see CP nodes)

Reply:
nevermind, I see your note 👍

Reply:
Good shout 🤔 But well, this variable is ignored if `wait_for_total_control_plane_nodes` is set to `false` (the default), so I'd say that for EKS we don't need to set anything / do anything different, and things will just work.